adapters/pubmatic.go

```diff
@@ -56,7 +56,12 @@ func (a *PubmaticAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder
 		}
 		pbReq.Imp[i].Banner.Format = nil // pubmatic doesn't support
 		pbReq.Imp[i].TagID = params.AdSlot
-		pbReq.Site.Publisher = &openrtb.Publisher{ID: params.PublisherId}
+		if pbReq.Site != nil {
+			pbReq.Site.Publisher = &openrtb.Publisher{ID: params.PublisherId}
+		}
```

Comment:
The pbReq object is created in openrtb_util.go, and it's explicitly either Site or App. Do you think it's worth re-validating that here?